<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Jobsdatabase</class>
 <widget class="QMainWindow" name="Jobsdatabase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Job's Database</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>LogoDefault.png</normaloff>LogoDefault.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>250</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string notr="true">Jobs:</string>
    </property>
   </widget>
   <widget class="QListView" name="list">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>250</width>
      <height>500</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="add_job">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>120</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">Add Job</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="new_job_text">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>80</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="plainText">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_job">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>180</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">Delete Selected Job</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_all">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>240</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">Delete All Jobs</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
